zipkin-web/src/main/resources/app/js/component_ui/serviceName.js

```diff
@@ -2,16 +2,17 @@
 
 define(
   [
-    'flight/lib/component'
+    'flight',
+    'js-cookie',
+    'chosen'
   ],
 
-  function (defineComponent) {
-
-    return defineComponent(serviceName);
+  function (flight, Cookies, chosen) {
+    return flight.component(serviceName);
 
     function serviceName() {
       this.onChange = function() {
-        $.cookie('last-serviceName', this.$node.val());
+        Cookies.set('last-serviceName', this.$node.val());
```

Comment:
jquery-cookie is replace with js-cookie: https://github.com/carhartl/jquery-cookie